cmd/site-replication.go

```diff
@@ -1484,8 +1575,10 @@ func (c *SiteReplicationSys) syncLocalToPeers(ctx context.Context) SRError {
 			if err != nil {
 				return errSRBackendIssue(err)
 			}
-			if _, isLDAPAccount := claims[ldapUserN]; !isLDAPAccount {
-				continue
+			if claims != nil {
+				if _, isLDAPAccount := claims[ldapUserN]; !isLDAPAccount {
+					continue
+				}
```

Comment:
should we error out if `claims` don't have ldapUserN ? - since that is not supported yet?

Reply:
Since the first site can have data prior to configuring site replication, the intention here is to skip non LDAP service accounts created before LDAP was set up. 

This change was just to avoid a panic observed in the logs when claims was nil, nothing specific to adding new site

Reply:
okay @poornas

Reply:
adding some comments would help